Makefile

```diff
@@ -27,6 +27,9 @@ virtualenv: $(PYTHON)
 $(PYTHON):
 	$(VIRTUALENV) $(VENV)
 
+set-data:
+	if [ ! -d "data" ]; then mkdir data; cp -a -v example-data/. data; fi
+
```

Comment:
If you only do that, then you'll have to call `make set-data` manually. 

You can do all this using the Makefile mechanics:

```
data:
    cp -R example-data/ data/

tests:  install-dev data
    ...
```

With this, when you run `make tests`, `make` automatically checks for the existence of `data`